coherence/www/assets/app.js

```diff
@@ -0,0 +1,139 @@
+"use strict";
+
+let io = new IO();
+
+// Prepare elements from DOM
+let _td_features = {};
+const columns = ["bpm", "lf", "hf", "lf/hf"]; // columns of EMG activation bursts
+columns.forEach(function (column) {
+  _td_features[column] = document.getElementById(column);
+});
+const td_bpm = document.getElementById("bpm");
+const _gradientColors = [
```

Comment:
Would be great to report the colors in the README or at least to provide a succinct explanation

Reply:
Great. I would avoid mentioning Gandhi though (here and in the code).